app/src/main/java/com/example/image_search/CaptureImageActivity.java

```diff
@@ -122,9 +122,19 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             int columnIndex = cursor.getColumnIndex(filePathColumn[0]);
             imagePath = cursor.getString(columnIndex);
             cursor.close();
+
+            // open crop screen
```

Comment:
the comments message doesn't represent what the code does. Make it clean, please 👍